<<<<<<< HEAD
#Service to set digital outputs on/off
int64 DIO_num  #digital IO number
bool state  #1-on; 0-off
---
int64 ret
=======
#Service to set digital outputs on/off
int64 DIO_num  #digital IO number
bool state  #1-on; 0-off
---
int64 ret
>>>>>>> 1b1f87b6247333221ca1660d810eda01b82fd03f
string msg